### ISSUE TITLE
hold command|alt spacebar Siri shortcut

### ISSUE THREAD

**Issue by rkusa** [tekezo/Karabiner-Elements#36]
With Karabiner-Elements active, the hold command|alt spacebar shortcuts for Siri in macOS Sierra are not working. This is not a big deal, but maybe still worth the bug report.

<img width="670" alt="screen shot 2016-09-21 at 08 25 56" src="https://cloud.githubusercontent.com/assets/409021/18700013/0d9f2442-7fd5-11e6-9c3e-4b9225628f67.png">

Thanks for the effort of getting Karabiner to work with macOS Sierra!

**Comment by tekezo** [comment]
Thank you for feedback!
I've fixed this issue at v0.90.33.

Please upgrade to the latest version from "Misc > Check for updates" or 
https://pqrs.org/latest/karabiner-elements-latest.dmg

**Comment by rkusa** [comment]
Thanks for the fix! I found a remaining strange behaviour. After having installed the update (was v0.90.34 now), it did not work. I rebooted, after which it worked. However, once I changed the shortcut again (e.g. from hold command space to hold option space), it stopped working again (even if I change back to hold command space). Let me know if there is anything I can provide you that helps narrowing down this behaviour.
